src/test/java/com/gravity9/mongocdc/ChangeStreamTest.java

```diff
@@ -5,33 +5,43 @@
 import com.mongodb.client.model.changestream.ChangeStreamDocument;
 import com.mongodb.client.model.changestream.FullDocument;
 import com.mongodb.client.model.changestream.OperationType;
-import java.util.List;
-import java.util.Map;
+import com.mongodb.client.result.InsertOneResult;
 import org.bson.Document;
 import org.bson.types.ObjectId;
 import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
 
-import static com.gravity9.mongocdc.constants.TestConstants.COLL_NAME;
-import static com.gravity9.mongocdc.constants.TestConstants.CONN_URI;
-import static com.gravity9.mongocdc.constants.TestConstants.DB_NAME;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class ChangeStreamTest {
+public class ChangeStreamTest extends AbstractMongoDbBase {
+
+	private static final Logger log = LoggerFactory.getLogger(ChangeStreamTest.class);
+
+	@BeforeEach
+	public void setup() {
+		super.setup();
+		mongoConfig = MongoConfig.builder()
+				.connectionUri(getConnectionUri())
+				.databaseName(getDatabaseName())
+				.collectionName(getTestCollectionName())
+				.workerConfigCollectionName("changeStreamWorkerConfig")
```

Comment:
Use `getWorkerConfigCollectionName()` and `getClusterConfigCollectionName()` methods

Reply:
Fixed